docs/pages/push-notifications/receiving-notifications.md

```diff
@@ -76,6 +76,11 @@ export default class App extends React.Component {
 - `shouldPlaySound`
 - `shouldSetBadge`
 
+
+## Closed Notification Behavior Android
+
+**Important Note**: Due to certain battery optimisation settings from several android vendors, the operating system may prevent background tasks when the app is **closed**, preventing notifications being delivered.
+
```

Comment:
```suggestion
## Closed Notification Behavior

On Android, users can set certain OS-level settings that can prevent notifications from being delivered. One such setting is the "Deep Clear" option on OnePlus devices.

```

Reply:
I think the message should specify that this only affects delivery when the app is closed as otherwise other delivery issues could be confused with this.

I also think that it should be mentioned that these settings usually revolve around increased performance / battery optimisation to prompt others to look in the right places.

Is there any reasoning behind removing "Android" to the header? I would have thought this would have brought users to attention better?

What is your thoughts on potentially adding a list of settings known to cause the issue across different devices? It could get long and unwieldy but could be useful, more a question of style I think.


```suggestion
## Closed Notification Behaviour Android

On Android, users can set certain OS-level settings revolving around performance and battery optimisation, that can prevent notifications from being delivered. One such setting is the "Deep Clear" option on OnePlus devices.

```

Reply:
> I think the message should specify that this only affects delivery when the app is closed as otherwise other delivery issues could be confused with this.

Sure we can add that in

> I also think that it should be mentioned that these settings usually revolve around increased performance / battery optimisation to prompt others to look in the right places.

I'm not sure that this is true of most of these settings, which is why I hesitate to add that phrase in here

> Is there any reasoning behind removing "Android" to the header?

consistency with the header above- `Foreground Notification Behavior`, and if there are other settings we find on iOS later it's easy to add to this section

> What is your thoughts on potentially adding a list of settings known to cause the issue across different devices?

I don't think that the list of settings should be included here. It would be hard to keep it accurate and up-to-date

Reply:
> I'm not sure that this is true of most of these settings, which is why I hesitate to add that phrase in here

I can only speak about my particular experience first hand, however the "Free-code-camp" article has the following to say:

> It should also be noted that the terminology used by each manufacturer is different. The auto-launch feature can be referred to as app auto-start, start-up manager, auto-start manager, app optimisation, protected apps, or background app management.

This is also why I added "usually" but perhaps this could be changed to be put in bold?


```suggestion
## Closed Notification Behaviour

On Android, users can set certain OS-level settings (**usually** revolving around performance and battery optimisation), that can prevent notifications from being delivered when the app is closed. One such setting is the "Deep Clear" option on OnePlus devices.

```